cin_validator/__main__.py

```diff
@@ -183,5 +186,16 @@ def cli_converter(filename: str):
         click.echo(f"{filename} can't be found, have you entered it correctly?")
 
 
+@cli.command(name="timer")
+def timer():
+    st = datetime.datetime.now()
+    os.system(
+        "python -m cin_validator run-all /workspaces/CIN-validator/fake_data/fake_CIN_data.xml"
+    )
+    et = datetime.datetime.now()
+    time_elapsed = et - st
+    print(f"Time to run: {time_elapsed}")
+
```

Comment:
This is useful.

Did you already consider allowing the filepath to be passed in as a command line argument?.

Reply:
Hi @tab1tha I did, I just realised we'd likely be using the same sample data every time, so there was no real reason to allow the filepath to be added as a CLI argument. I could potentially add it as an optional argument? I was going to do that originally actually but I was having a hard time getting one CLI function to work within this one.

Reply:
Your filepath is a codespaces path and mine isn't so we'll still have to swap it out each time we run.

How about passing it in as a string and then appending that string to the `"python -m cin_validator run-all` string?
You could also use  a formatted-string format to put them together.

In any case, if this will take more than a moment to resolve, add a TODO statement above it and let it be. We can get to it later. There's a stack of tasks to prioritise.

Reply:
let's add it as a todo and come back later